group_vars/maintenance.yml

```diff
@@ -117,6 +117,8 @@ fsm_scripts:
       - /data/jwd/tmp
       - /data/jwd02f/tmp
       - /data/jwd04/tmp
+      - "{{ upload_dir_main }}"
```

Comment:
Shouldn't we add this to the `upload_dirs` below?

Reply:
good point :D